### PR TITLE
BFD for OSPF

### PR DESCRIPTION
**Pull request by MarkStitson**
This is the code for enabling BFD support in the UI for  OSPF. It is based on the same code that did this for BGP as discussed in Issue 2050 ( https://github.com/opnsense/plugins/issues/2050 ).

I tested that the config files get generated correectly and @jberkleef test the code in his setup as per issue.

### CODE REVIEW

File: net/frr/src/opnsense/service/templates/OPNsense/Quagga/ospf6d.conf [hunk 1]
```diff
@@ -25,6 +25,9 @@ agentx
 {%   for interface in helpers.toList('OPNsense.quagga.ospf6.interfaces.interface') %}
 {%     if interface.enabled == '1' %}
 interface {{ physical_interface(interface.interfacename) }}
+{%        if 'bfd' in interface and interface.bfd == '1' %}
```
Comment (AdSchellevis) [inline net/frr/src/opnsense/service/templates/OPNsense/Quagga/ospf6d.conf:28]: The missing tag is easier solved using the following construct:
```
if interface.bfd|default('') == '1'
```

**Comment by MarkStitson** [comment]
I've added the suggested improvement to my code ospf code and to the original bgp code.

**Comment by AdSchellevis** [comment]
@mimugmail are you ok with merging this? it does look ok to me

**Comment by mimugmail** [comment]
Yes, looks good. Thx :)

**Comment by AdSchellevis** [comment]
@MarkStitson thanks for contributing!

**Pull request merged**